src/diffpy/pdfgui/gui/temperatureseriespanel.py

```diff
@@ -288,18 +300,26 @@ def onDelete(self, event):  # wxGlade: TemperatureSeriesPanel.<event_handler>
         self.fillList()
         return
 
+    def create_filename_order(self):
+        filenames = [f for t, f in self.datasets]
+        numericStringSort(filenames)
+        self.filename_order = dict(zip(filenames, range(len(filenames))))
+        return
+
+    def filename_sortkey(self, tf):
```

Comment:
when we do turn lambdas into functions, mostly they should be private functions (just put an underscore at the beginning of the name).